specs/001-kiro-specs-knowledge/research.md

```diff
@@ -0,0 +1,96 @@
+# Research: Knowledge Graph Systems for NWB Data
+
+## RDF Triple Store Selection
+
+**Decision**: Use rdflib with optional persistent backend (SQLite/PostgreSQL)
```

Comment:
I don't want to see SQLite anywhere. PostgreSQL all the way. Transactions with rollback during testing in a local docker compose stack. Supporting both is not worth it. We want to avail of the best of Postgres